chef/data_bags/crowbar/migrate/crowbar/301_add_client_password.rb

```diff
@@ -0,0 +1,11 @@
+def upgrade(template_attrs, template_deployment, attrs, deployment)
```

Comment:
Maybe we do not plan to backport it, but to be sure, make this work for upgrades and check for the presence of the keys...

Reply:
Good point, this will have to be backported to stable/5.0-pike

Reply:
Done